Core/ModuleInstaller.cs

```diff
@@ -889,29 +889,41 @@ private void Uninstall(string modName)
         /// <summary>
         /// Takes a collection of directories and adds all parent directories within the GameData structure.
         /// </summary>
-        /// <param name="directories">The collection of directory path strings to examine </param>
-        private HashSet<string> AddParentDirectories(HashSet<string> directories )
+        /// <param name="directories">The collection of directory path strings to examine</param>
+        public HashSet<string> AddParentDirectories(HashSet<string> directories)
         {
-            var newDirectories = new HashSet<string>();
-            foreach (string directory in directories)
+            if (directories == null || directories.Count == 0)
             {
-                if ( directory.Contains(ksp.GameData()))
+                return new HashSet<string>();
+            }
+
+            var gameData = KSPPathUtils.NormalizePath(ksp.GameData());
```

Comment:
This becomes KSPPathUtils.NormalizePath(KSPPathUtils.NormalizePath( Path.Combine(GameDir(), "GameData")))